lib/gengi.test.js

```diff
@@ -1,10 +1,91 @@
 import { expect } from 'chai'
-import gengi from './gengi'
+import Gengi from './gengi'
 
-describe('Gengi utility', () => {
+describe('lib/gengi - unit', () => {
   it('should be a function', () => {
-    expect(gengi).to.be.a('function')
+    expect(Gengi).to.be.a('function')
```

Comment:
I wish this line could be `expect(Gengi).to.be.a('class')` but I'm expecting it to be a drawback from transpiling?